Dockerfile

```diff
@@ -60,13 +60,27 @@ COPY . /app/
 RUN pip uninstall -y ashley
 RUN pip install -e .[dev]
 
+# Install dockerize. It is used to ensure that the database service is accepting
+# connections before trying to access it from the main application.
+ENV DOCKERIZE_VERSION v0.6.1
```

Comment:
In lot of projects we choose to remove dockerize from the application image and use directly their image in the `docker-compose` file. Take a look at [marsha's](https://github.com/openfun/marsha/blob/31700e7536417c44fd3ed14354ad21db270467a7/docker-compose.yml) `docker-compose.yml` file